git-remote-hg

```diff
@@ -265,10 +266,10 @@ class Parser:
         return (user, int(date), -tz)
 
 def fix_file_path(path):
-    path = os.path.normpath(path)
+    path = posixpath.normpath(path)
     if not os.path.isabs(path):
```

Comment:
@alexei-31 uhm, sorry, I do not understand a word of what you are saying. But it's good that you mention razaraz , I did not realize the commit was by him :).

@razaraz Could you perhaps briefly explain whether it is intentional that the `isabs` call is not converted (to be frank, I am not even sure right now why it is there in the first place, even on POSIX systems -- I'd have to drill down into the code again first; so my hope is that you did this intentionally and can explain, thus saving some time and efforts :-).